sdk/purview/purview-datamap-rest/src/outputModels.ts

```diff
@@ -668,7 +668,7 @@ export interface SearchResultValueOutput {
   /** The description of the asset. */
   description?: string;
   /** The endorsement of the asset. */
-  endorsement?: string;
+  endorsement?: string[];
```

Comment:
Is this expected for the url template change? @MaryGao

Reply:
this is introduced by spec change not any feature in url template: https://github.com/Azure/azure-rest-api-specs/pull/33310/files#diff-0b59e3de4df64d4d8a0646b8631146f2f4b67913bb4efd4fc4e62be3cf46e02c.